packages/data/tests/unit/adapters/metadata/bard-test.ts

```diff
@@ -0,0 +1,78 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import BardMetadataAdapter from 'navi-data/adapters/metadata/bard';
+//@ts-ignore
+import { setupMirage } from 'ember-cli-mirage/test-support';
+
+let Adapter: BardMetadataAdapter;
+
+module('Unit | Adapter | metadata/bard', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function(this: TODO) {
+    Adapter = this.owner.lookup('adapter:metadata/bard');
+  });
+
+  test('_buildURLPath', function(assert) {
+    assert.equal(
+      Adapter['buildURLPath']('table', '', {}),
```

Comment:
```suggestion
      Adapter.buildURLPath('table', '', {}),
```

Reply:
@kevinhinterlong `buildURLPath` is private. Not sure if there is a better way to do this.